test/unit/contract_test.js

```diff
@@ -59,7 +62,11 @@ describe('Contract', function () {
       StellarBase.nativeToScVal(2, { type: 'i32' })
     );
 
-    xit('builds valid XDR', function () {
+    it('works with no parameters', function () {
+      contract.call('empty').toXDR();
```

Comment:
not now, but would the `toXDR()` output need to be compared with expected output? or if it doesn't throw at that point is good enough.

Reply:
yeah the structure itself is checked in other tests, this is just a "did you build working xdr" test (helps check types, etc. when it isn't marked to skip :facepalm:)